entries/w/wisc.edu.json

```
@@ -0,0 +1,17 @@
+{
+    "University of Wisconsin Madison": {
+        "domain": "wisc.edu",
+        "url": "https://www.wisc.edu/",
+        "tfa": [
+            "custom-software",
+            "custom-hardware"
```

According to https://kb.wisc.edu/helpdesk/90562, U2F keys are also supported.
```suggestion
            "custom-software",
            "custom-hardware",
            "u2f"
```